src/SideBar.jsx

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import { useState } from 'react';
+import SideBarCard from './SideBarCard';
+import './SideBar.css';
+
+const SideBar = (props) => {
+    const likedMovies = props.likedList;
```

Comment:
typically we like to destructure the props to access them:
`const {likedList, watchedList, showing, etc} = props;`